enterprise/helpers.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+"""
+Helper functions for enterprise app.
+"""
+
+from .models import EnterpriseCustomerBrandingConfiguration
```

Comment:
@zubair-arbi Ok, well if people want to go for absolute imports we should turn on a pylint rule to enforce it; otherwise some people won't know or will forget.